plugin.xml

```diff
@@ -55,7 +55,7 @@
             </feature>
         </config-file>
         <config-file target="AndroidManifest.xml" parent="/*">
-            <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+            <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" android:maxSdkVersion="32" />
         </config-file>
```

Comment:
I guess you should add here also 

```xml
<uses-permission android:name="android.permission.READ_MEDIA_IMAGES" android:minSdkVersion="33" />
```

based on suggestion from @felicienfrancois

Reply:
@ochakov could you add that to this PR, such that I can use this PR in my project?
thanks

Reply:
Made the following changes in this PR:
1. Video permission when taking video instead of image permission as requested by @breautek 
2. Updated manifest as requested by @jfoclpf